scripts/linux/fv/gitlab_test_fv.sh

```diff
@@ -53,7 +53,7 @@ do
 	# Run functional tests
 	${FV_HOME}/gitlab-olp-cpp-sdk-functional-test.sh
 
-	if [[ $? -eq 1 ]]; then
+	if [[ $? -ne 0 ]]; then
```

Comment:
I don't think this is correct.
Once a test crashes there is no reason why to repeat it multiple times, it is just a waste of CI time and developer time who needs to scroll for half an hour to find the crash in the log.
In my opinion you should add 3 checks:

```
if [[ $? -eq 1 ]]; then
  # Handle flacky test and repeat up to 3 times
elif  [[ $? -gt 1 ]]; then
  # Crash happened -> exit 1
else
  # Success -> break
```

Reply:
Good advice, thanks!